Sources/SFSafeSymbols/Symbols/SFSymbol+3.0.swift

```diff
@@ -4,2495 +4,6543 @@
+    ///
+    /// Layersets:
+    /// - Monochrome
+    ///
+    /// - Warning: ⚠️ This symbol may not be modified and may only be used to refer to Apple’s AirPods.
     static let airpodsChargingcaseWirelessFill = SFSymbol(rawValue: "airpods.chargingcase.wireless.fill")
 
     /// 􀹫
-    /// ⚠️ This symbol may not be modified and may only be used to refer to Apple’s AirPods Pro.
+    /// Single Localization, Single Layerset, ⚠️ Restricted
+    ///
+    /// Layersets:
+    /// - Monochrome
+    ///
+    /// - Warning: ⚠️ This symbol may not be modified and may only be used to refer to Apple’s AirPods Pro.
     static let airpodsproChargingcaseWireless = SFSymbol(rawValue: "airpodspro.chargingcase.wireless")
 
     /// 􀹬
-    /// ⚠️ This symbol may not be modified and may only be used to refer to Apple’s AirPods Pro.
+    /// Single Localization, Single Layerset, ⚠️ Restricted
+    ///
+    /// Layersets:
+    /// - Monochrome
+    ///
+    /// - Warning: ⚠️ This symbol may not be modified and may only be used to refer to Apple’s AirPods Pro.
     static let airpodsproChargingcaseWirelessFill = SFSymbol(rawValue: "airpodspro.chargingcase.wireless.fill")
 
     /// 􁄾
-    /// ⚠️ This symbol may not be modified and may only be used to refer to Apple’s AirTag.
+    /// Single Localization, Single Layerset, ⚠️ Restricted
+    ///
+    /// Layersets:
+    /// - Monochrome
+    ///
+    /// - Warning: ⚠️ This symbol may not be modified and may only be used to refer to Apple’s AirTag.
     static let airtag = SFSymbol(rawValue: "airtag")
 
     /// 􁄿
-    /// ⚠️ This symbol may not be modified and may only be used to refer to Apple’s AirTag.
+    /// Single Localization, Single Layerset, ⚠️ Restricted
+    ///
+    /// Layersets:
+    /// - Monochrome
+    ///
+    /// - Warning: ⚠️ This symbol may not be modified and may only be used to refer to Apple’s AirTag.
     static let airtagFill = SFSymbol(rawValue: "airtag.fill")
 
     /// 􁄼
-    /// From iOS 15.0, macOS 12.0, tvOS 15.0 and watchOS 8.0 on, the following localizations are available: Right-To-Left
-    /// ⚠️ This symbol may not be modified and may only be used to refer to Apple’s AirTag.
+    /// 2 Localizations, 2 Layersets, ⚠️ Restricted
+    ///
+    /// Localizations:
+    /// - Latin
+    /// - Right-To-Left
+    ///
+    /// Layersets:
+    /// - Monochrome
+    /// - Hierarchical
+    ///
+    /// - Warning: ⚠️ This symbol may not be modified and may only be used to refer to Apple’s AirTag.
     static let airtagRadiowavesForward = SFSymbol(rawValue: "airtag.radiowaves.forward")
 
     /// 􁄽
-    /// From iOS 15.0, macOS 12.0, tvOS 15.0 and watchOS 8.0 on, the following localizations are available: Right-To-Left
-    /// ⚠️ This symbol may not be modified and may only be used to refer to Apple’s AirTag.
+    /// 2 Localizations, 2 Layersets, ⚠️ Restricted
+    ///
+    /// Localizations:
+    /// - Latin
+    /// - Right-To-Left
```

Comment:
I noticed that Apple changes the default localization to "Left-to-Right" when the only other localization is RTL, it would be nice if we could do the same.

Reply:
Good idea! I implemented it :)